packages/vscode-extension/syntaxes/cypher.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
This is generated automatically from the `textMateGrammar.ts` using `genTextMate.js`